examples/passport-login/data/db.json

```diff
@@ -1,13 +1,15 @@
 {
   "ids": {
-    "User": 1,
-    "UserIdentity": 1,
+    "User": 2,
+    "UserIdentity": 3463934540288187,
     "UserCredentials": 1
   },
   "models": {
     "User": {
+      "1": "{\"name\":\"Deepak Rajamohan\",\"username\":\"deepak.r.kris@gmail.com\",\"email\":\"deepak.r.kris@gmail.com\",\"id\":1}"
     },
     "UserIdentity": {
+      "3463934540288186": "{\"id\":\"3463934540288186\",\"provider\":\"facebook\",\"profile\":{\"emails\":[{\"value\":\"deepak.r.kris@gmail.com\"}]},\"authScheme\":\"facebook\",\"created\":\"2020-05-06T00:48:12.355Z\",\"userId\":1}"
```

Comment:
I remember that we had a discussion not to include `db.json` as the data change over time.

Reply:
yes, I did include in the git ignore, not sure how this gets included every time, I will fix that as well.

Reply:
i just removed the file altogether and its reference